docs/getting_started/using_modin/using_modin_cluster/using_modin_ray_cluster.rst

```diff
@@ -0,0 +1,125 @@
+================================
+Using Modin in a AWS Ray Cluster
+================================
+
+.. note::
+  | *Estimated Reading Time: 15 minutes*
+
+Often in practice we have a need to exceed the capabilities of a single machine.
+Modin works and performs well in both local mode and in a cluster environment.
+The key advantage of Modin is that your notebook does not change between
+local development and cluster execution. Users are not required to think about
+how many workers exist or how to distribute and partition their data;
+Modin handles all of this seamlessly and transparently.
+
+.. image:: ../../../img/modin_cluster.png
+   :alt: Modin cluster
+   :align: center
+   :scale: 90%
+
+Extra requirements for AWS authentication
+-----------------------------------------
+
+First of all, install the necessary dependencies in your environment:
+
+.. code-block:: bash
+
+   pip install boto3
+
+The next step is to setup your AWS credentials. One can set  `AWS_ACCESS_KEY_ID`, 
+`AWS_SECRET_ACCESS_KEY` and `AWS_SESSION_TOKEN` environment variables or  
+just run the following command:
+
+.. code-block:: bash
+
+   aws configure
+
+Starting and connecting to the cluster
+--------------------------------------
+
+This example starts 1 head node (m5.24xlarge) and 5 worker nodes (m5.24xlarge), 576 total CPUs.
+You can check the `Amazon EC2 pricing`_ .
+
+You can manually create AWS EC2 instances and configure them or just use the `Ray autoscaler` to 
```

Comment:
Can we point to Ray autoscaler docs here instead of just refering to it as `Ray autoscaler`?

Reply:
Ok, I rewrote it to "Ray CLI" as written in the Ray documentation.